user-org-actor/src/main/java/org/sunbird/user/dao/UserOpenSaberDaoImpl.java

```diff
@@ -0,0 +1,51 @@
+package org.sunbird.user.dao;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import io.opensaber.registry.helper.RegistryHelper;
+import org.apache.commons.lang3.StringUtils;
+import org.sunbird.Application;
+import org.sunbird.exception.BaseException;
+import org.sunbird.exception.ProjectCommonException;
+import org.sunbird.exception.message.IResponseMessage;
+import org.sunbird.exception.message.Localizer;
+import org.sunbird.exception.message.ResponseCode;
+import org.sunbird.response.Response;
+import org.sunbird.util.ProjectLogger;
+import org.sunbird.util.jsonkey.JsonKey;
+
+import java.util.Map;
+
+/**
+ * @author Amit Kumar
+ * This class will contains method to interact with open saber
+ */
+public class UserOpenSaberDaoImpl implements IUserDao {
+
+    private Localizer localizer = Localizer.getInstance();
+    private RegistryHelper registryHelper;
+    private ObjectMapper objectMapper;
+
+    public UserOpenSaberDaoImpl() {
+        objectMapper = new ObjectMapper();
+        registryHelper = Application.applicationContext.getBean(RegistryHelper.class);
+    }
+
+    @Override
+    public Response addUser(Map<String, Object> user) throws BaseException {
+        Response response = new Response();
+        try {
+            long startTime = System.currentTimeMillis();
+            String userId = registryHelper.addEntity(objectMapper.convertValue(user, JsonNode.class),"");
+            if (StringUtils.isNotBlank(userId)) {
```

Comment:
You must not get a blank userId here. Remove if block and trust the layer beneath will throw exception. Am concerned on branch coverage and possibilities of un-detected failure of code with these patterns.